public/assets/html/createCharacter.html

```diff
@@ -0,0 +1,101 @@
+            <li id="Defense" value="20">
+                <h4 class="skillValStr">Defense : 20</h4>
+                <button class='btn waves-effect waves-light grey darken-3 white-text' data-skill="Defense">-</button>
+                <button class='btn waves-effect waves-light grey darken-3 white-text' data-skill="Defense">+</button>
+            </li>
+            <li id="Attack" value="20">
+                <h4 class="skillValStr">Attack : 20</h4>
+                <button class='btn waves-effect waves-light grey darken-3 white-text' data-skill="Attack">-</button>
+                <button class='btn waves-effect waves-light grey darken-3 white-text' data-skill="Attack">+</button>
+            </li>
+            <li id="Health" value="20">
+                <h4 class="skillValStr">Health : 20</h4>
+                <button class='btn waves-effect waves-light grey darken-3 white-text' data-skill="Health">-</button>
+                <button class='btn waves-effect waves-light grey darken-3 white-text' data-skill="Health">+</button>
+            </li>
+        </ul>
+    </div>
+    <br>
+    <button class='btn btn-large waves-effect waves-light grey darken-3 white-text' id="submitBtn">submit</button>
+
+
+    <script>
+        let skillPonints = 20;
+        $("#skillPoints").text("Skill Points: " + skillPonints);
+
+        // increase decrease skill value
+        $(".skills button").on("click", function (event) {
+            event.preventDefault();
+            let skill = $(this).attr("data-skill");
+            let skillVal = parseInt($("#" + skill).attr("value"));
+            if ($(this).text() == "+") {
+                if (skillPonints > 0) {
+                    skillPonints--;
+                    skillVal++;
+                }
+            } else if ($(this).text() == "-") {
+                if (skillVal > 20) {
+                    skillPonints++;
+                    skillVal--;
+                }
+            }
+            $("#skillPoints").text("Skill Points: " + skillPonints);
+            $("#" + skill).attr("value", skillVal.toString());
+            $("#" + skill + " .skillValStr").text(skill + " : " + skillVal.toString());
+        });
+
+        $("#submitBtn").on("click", function (event) {
+            event.preventDefault();
+            let newChar = {
+                "name": $("#name").val().trim(),
+                "attack": parseInt($("#Attack").attr("value")),
+                "defense": parseInt($("#Defense").attr("value")),
+                "health": parseInt($("#Health").attr("value")),
+            }
+            console.log(newChar);
+            createChar(newChar.name, newChar.attack, newChar.defense, newChar.health);
+        });
+        function createChar(name, attack, defense, health) {
+            $.post("/api/create", {
+                name: name,
+                attack: attack,
+                defense: defense,
+                health: health
+            }).then(() => {
```

Comment:
1 line this and drop braces since you don't return a value
.then(() => location.reload());